### ISSUE TITLE
ggplot() works only with a limited number of themes, throws fatal error otherwise

### ISSUE THREAD

**Issue by DudeMcGee** [tidyverse/ggplot2#4309]
Problem: I cannot use all the themes I theoretically should be able to use.
This works:
```r
ggplot()+theme_calc()+geom_line(aes(y=value, x=run), data = collatzdata)
```
But this doesn't:
```r
ggplot()+theme_base()+geom_line(aes(y=value, x=run), data = collatzdata)
```
It throws this error:
```
Error in grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) : 
  X11 font -adobe-helvetica-%s-%s-*-*-%d-*-*-*-*-*-*-*, face 1 at size 16 could not be loaded
```
I have installed every single X font I could find in the world. But nothing helped. Also the systemfonts package has no use.

I have experienced this problem on a completely fresh installed Kubuntu 20.10 Desktop virtual machine as well as on a laptop with the same OS. Out of curiosity I have installed Anaconda3 with RStudio which worked perfectly right from the start. But I have uninstalled it completely again and set up R and RStudio from scratch after Anaconda3 was gone again. I especially wonder why adobe and helvetica are hardcoded somewhere in the source. I can't help considering this as the cause of the problem. But I'm not an experienced programmer, so...

**Comment by thomasp85** [comment]
This sounds like a device issue. Can you try to reproduce the error using the ragg devices (e.g. `ragg::agg_png()`)

**Comment by DudeMcGee** [comment]
I have downloaded and successfully compiled the necessary libraries to get ragg running. I also did several restarts of RStudio, which is sometimes necessary to get to updating the package list and so on.
In my most recent test session it made no difference when ragg was active and attached or not. Unfortunately the code example I found for agg_png() did not work:

```
agg_png(
  filename = "Rplot.png",
  width = 480,
  height = 480,
  units = "px",
  pointsize = 12,
  background = "white",
  res = 72,
  scaling = 1,
  bitsize = 8
)
```
This command had simply no effect at all.
